examples/read_bids_datasets.py

```diff
@@ -92,17 +93,24 @@
 # For now, we're interested only in the EEG data in the BIDS root directory
 # of the Parkinson's disease patient dataset. There were two sessions, one
 # where the patients took their regular anti-Parkinsonian medications and
-# one where they abstained for more than twelve hours. Let's start with the
-# off-medication session.
+# one where they abstained for more than twelve hours. For now, we are
+# not interested in the on-medication session.
 
+sessions = get_entity_vals(bids_root, 'session', ignore_sessions='on')
 datatype = 'eeg'
-session = 'off'
-bids_path = BIDSPath(root=bids_root, session=session, datatype=datatype)
+extensions = [".bdf", ".tsv"]  # ignore .json files
+bids_paths = find_matching_paths(bids_root, datatypes=datatype,
+                                 sessions=sessions, extensions=extensions)
 
 # %%
 # We can now retrieve a list of all MEG-related files in the dataset:
+print(bids_paths)
 
-print(bids_path.match())
+# %%
+# Note that this is the same as running:
+session = 'off'
+bids_path = BIDSPath(root=bids_root, session=session, datatype=datatype)
+print(bids_path.match(ignore_json=True))
```

Comment:
Should we print the same list twice to show that the result is the same or should we comment out 
```Python
# print(bids_path.match(ignore_json=True))
``` 
and just say that this would yield the same?

I went for printing it twice to make it as obvious as possible.